static/js/portico/help.js

```diff
@@ -17,10 +17,42 @@ function registerCodeSection($codeSection) {
     $li.eq(0).click();
 }
 
+function adjust_mac_shortcuts() {
+    var keys_map = new Map([
+        ['Backspace', 'Delete'],
+        ['Enter', 'Return'],
+        ['Home', 'Fn + ⇽'],
+        ['End', 'Fn + ⇾'],
+        ['PgUp', 'Fn + ↑'],
+        ['PgDn', 'Fn + ↓'],
+    ]);
+
+    $(".markdown .content code").each(function () {
+        var text = $(this).text();
+
+        if (!keys_map.has(text)) {
+            return;
+        }
+
+        var key_string = keys_map.get(text);
+        var keys = key_string.match(/[^\s\+]+/g);
+
+        _.each(keys, function (key) {
+            key_string = key_string.replace(key, '<code>' + key + '</code>');
```

Comment:
Why is there `<code>` on the right hand side, but not the left?

Also, for the match we shouldn't match against this without the backticks or `<code>`, e.g. otherwise it might match "Go to the Home view", or "Enter your 9 digit phone number".

Reply:
`$(".markdown .content code")` is responsible for only matching inline code blocks.

`key_string = key_string.replace(key, '<code>' + key + '</code>');` wraps the matched key in code tags.